src/kirby/components/floating-action-button/floating-action-button.component.tns.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, NgZone, Input } from '@angular/core';
+import { OrientationChangedEventData } from 'tns-core-modules/application';
+import { ContentView } from 'tns-core-modules/ui/content-view';
+import { EventData, View } from 'tns-core-modules/ui/page/page';
+import * as app from 'tns-core-modules/application';
+import { Color } from 'tns-core-modules/color';
+import { registerElement } from 'nativescript-angular';
+
+const FLOATING_ACTION_BUTTON_COMPONENT_SELECTOR = 'kirby-floating-action-button';
+
+declare const CGSizeMake: any;
+declare const android: any;
+declare var require: any;
+const style: any = require('sass-extract-loader!./floating-action-button.component.scss');
+
+@Component({
+  selector: FLOATING_ACTION_BUTTON_COMPONENT_SELECTOR,
+  templateUrl: './floating-action-button.component.html',
+  styleUrls: ['./floating-action-button.component.scss'],
+})
+export class FloatingActionButtonComponent extends ContentView {
+  @Input() iconSrc?: string = '~/assets/icons/add/add@3x.png';
+  @Input() hasShadow?: boolean = true;
+  @Input() disabled?: boolean = false;
+
+  view: View;
+
+  constructor(private zone: NgZone) {
+    super();
+  }
+
+  onViewLoaded(args: EventData) {
+    this.view = <View>args.object;
+    this.setupOnOrientationChangeListener();
+    this.addShadow();
+  }
+
+  setupOnOrientationChangeListener() {
+    app.on(app.orientationChangedEvent, (args: OrientationChangedEventData) => {
+      // Run in the zone, to make sure Angular data binding is informed of this:
+      this.zone.run(() => this.addShadow());
+    });
+  }
+
+  addShadow(): void {
+    if (!this.hasShadow) {
+      return;
+    }
+
+    if (this.view.android) {
+      let nativeView = this.view.android;
+      var shape = new android.graphics.drawable.GradientDrawable();
+      shape.setShape(android.graphics.drawable.GradientDrawable.OVAL);
+      shape.setColor(android.graphics.Color.parseColor(this.getThemeColor('kirby-primary-shade')));
+      // shape.setColor(android.graphics.Color.parseColor('#00ffa4')); // A brigther shadow color that prevents the darker borders around the shape on android
```

Comment:
Just remove this commented out line of code, you can always find it (again) in git

Reply:
completely agree.